packages/cache-core/src/strategies/fileSystem.ts

```diff
@@ -60,7 +60,7 @@ export class FileSystemCache implements CacheStrategy {
   async deleteAllByKeyMatch(pageKey: string, ctx: CacheContext) {
     if (!existsSync(ctx.serverCacheDirPath)) return
 
-    const pathToCacheFolder = path.join(ctx.serverCacheDirPath, pageKey)
+    const pathToCacheFolder = path.join(ctx.serverCacheDirPath, ...pageKey.split('/'))
```

Comment:
what is the reason for that?

Reply:
in the case when we have /page/page/ as a pageKey, and we have to prepare the path for this folder using path.join
for example for Windows OS delimiter is \ and path.join handles it